app/modules/jwplayer-assets.js

```diff
@@ -0,0 +1,47 @@
+import Ember from 'ember';
+const { RSVP } =  Ember;
+
+export const assetUrls = {
+	styles: '/mobile-wiki/assets/jwplayer/index.css',
+	script: '/mobile-wiki/assets/jwplayer/wikiajwplayer.js'
+};
+
+/**
+ * @class JWPlayerAssets
+ */
+class JWPlayerAssets {
+	constructor() {
+		this.wasStyleLoadInitialized = false;
+		this.scriptsPromise = null;
+	}
+
+	loadStyles() {
+		if (!this.wasStyleLoadInitialized) {
+			$(`<link rel="stylesheet" href="${assetUrls.styles}" crossorigin="anonymous">`)
+				.appendTo('head');
+			this.wasStyleLoadInitialized = true;
+		}
+	}
+
+	loadScripts() {
+		if (!this.scriptsPromise) {
+			this.scriptsPromise = new RSVP.Promise((resolve) => {
+				window.M.loadScript(assetUrls.script, true, (data) => {
+					resolve(data);
+				}, 'anonymous');
+			});
+		}
+
+		return this.scriptsPromise;
+	}
+
+	load() {
+		this.loadStyles();
+
+		return this.loadScripts();
+	}
+}
+
+const jwPlayerAssets = new JWPlayerAssets();
```

Comment:
Maybe we should create class with only static methods?

Reply:
As we talked offline, we'll leave it as is :)